bin/index.js

```diff
@@ -379,13 +372,13 @@ async function verifyCmd(packageName = null, options) {
 }
 
 async function hydrateCmd() {
-    deleteDB();
+    getVulnerabilityStrategy().deleteDB();
```

Comment:
Like depWalker.js the best here may to assign a vs variable.
```js
const vs = await getVulnerabilityStrategy();

await vs.deleteDB();
```

And same for hydrateDB (with vs).

Reply:
thats true actually